src/components/media-player/media-player.less

```diff
@@ -13,6 +13,13 @@
     padding: 0;
     height: 0;
   }
+
+  &.is-fullscreen {
+    display: grid;
+    grid-auto-flow: row;
+    align-items: center;
+    background-color: white;
```

Comment:
What do you think about `black` here? Especially when the video takes only half of the screen the big white margins feel a little overpowering to me (maybe I'm just used to having black margins from YouTube, but they seem to catch less attention).

Reply:
True, I agree, I've made it black 👍